submissions/Levi123/oop-exercise/index.js

```diff
@@ -0,0 +1,90 @@
+// class Animal {
+//    constructor (species, name, gender){
+//       this.species = species;
+//       this.name = name;
+//       this.gender = gender;
+//    }
+// }
+
+class Entity {
```

Comment:
`entity` is a generic term. Business domain and task requirements are a good source of naming ideas. Name should describe what entity it represents at a given level of abstraction - dog, cat, human, mammal, herbivore - in the given business domain.

Reply:
Done